src/main/java/com/devcycle/sdk/server/local/api/DVCLocalClient.java

```diff
@@ -1,35 +1,38 @@
-package com.devcycle.sdk.server.local;
+package com.devcycle.sdk.server.local.api;
 
-import com.devcycle.sdk.server.common.model.*;
+import java.util.Collections;
+import java.util.Map;
+
+import com.devcycle.sdk.server.common.model.Event;
+import com.devcycle.sdk.server.common.model.Feature;
+import com.devcycle.sdk.server.common.model.User;
+import com.devcycle.sdk.server.common.model.UserAndEvents;
+import com.devcycle.sdk.server.common.model.Variable;
```

Comment:
why did this change? can it just stay as `import com.devcycle.sdk.server.common.model.*;`?